src/Juvix/Compiler/Concrete/Translation/FromParsed/Analysis/Scoping/Error/Pretty.hs

```diff
@@ -10,11 +10,11 @@ import Juvix.Data.PPOutput
 import Juvix.Prelude
 import Text.EditDistance
 
-ppCode :: Scoped.PrettyCode c => c -> Doc Ann
-ppCode = runPP . Scoped.ppCode
+ppCode :: Scoped.PrettyCode c => Scoped.Options -> c -> Doc Ann
+ppCode opts = runPP opts . Scoped.ppCode
 
-runPP :: Sem '[Reader Scoped.Options] (Doc Scoped.Ann) -> Doc Ann
-runPP = code . run . runReader Scoped.defaultOptions
+runPP :: Scoped.Options -> Sem '[Reader Scoped.Options] (Doc Scoped.Ann) -> Doc Ann
```

Comment:
This should be removed. You have the Scoped.Options in the reader effect.

Reply:
The whole point of the runPP function is that runs the effect, so it needs concrete Scoped.Options to do this. Earlier it just used Scoped.defaultOptions

Reply:
Oh, that's right.